package.json

```diff
@@ -39,16 +39,14 @@
     "@backbase/positive-pay-check-v1-client-ang": "^1.3.0",
     "@backbase/positive-pay-journey-ang": "^0.44.0",
     "@backbase/select-context-widget-ang": "^4.4.0",
-    "@backbase/ui-ang": "^6.51.0",
+    "@backbase/ui-ang": "^7.5.0",
     "@backbase/universal-ang": "^7.0.0",
-    "@ng-bootstrap/ng-bootstrap": "^9.1.3",
-    "@ng-select/ng-select": "^6.1.0",
```

Comment:
not sure why ng-select and bignumber are removed. Is it expected?

Reply:
Those became direct dependencies of ui-ang, rather than peer deps AFAIK

Reply:
Yes, ui-ang now brings these in as regular deps, so they no longer need to be explicitly installed.

That said, I'm not sure _why_ they are now regular dependencies instead of peerDependencies.

This can actually cause build issues if your app project declares incompatible versions in its root package.json, as ui-ang can end up getting its own copy of these deep inside node_modules that doesn't get processed by `ngcc`, then you end up with the horrible `importing a module that does not have a Θmod property` error.

If this happens, npm doesn't give you any warnings or errors.  But if they were peerDependencies, then npm would warn/error if incompatible versions are declared in the app package.json.